tests/cql/CqlTypeOperatorsTest.xml

```diff
@@ -37,8 +37,8 @@
 			<output>@T14:30:00.000</output>
 		</test>
 		<test name="StringToDateTimeMalformed">
-			<expression invalid="true">convert '2014/01/01' to DateTime</expression>
-			<!-- EXPECT: Invalid format: "2014/01/01" is malformed at "/01/01" -->
+			<expression>convert '2014/01/01' to DateTime</expression>
+			<output>null</null>
```

Comment:
It appears the [XML CI](https://github.com/cqframework/cql-tests/actions/runs/9771598448/job/26974586849?pr=34) is failing because of the mismatched tag here

```suggestion
			<output>null</output>
```